contrib/openshift/skydive-template.yaml

```diff
@@ -82,9 +63,17 @@ objects:
         - args:
           - analyzer
           - --listen=0.0.0.0:8082
-          envFrom:
-          - configMapRef:
-              name: skydive-analyzer-config
+          env:
+          - name: SKYDIVE_ANALYZER_FLOW_BACKEND
+            value: elasticsearch
+          - name: SKYDIVE_ANALYZER_TOPOLOGY_BACKEND
+            value: elasticsearch
+          - name: SKYDIVE_ANALYZER_TOPOLOGY_PROBES
```

Comment:
Until now, we did not enable the k8s probe by default as it has some scalability issues.
A mitigation would be to limit the types of Kubernetes resources we are interested in through the "probes" section of the k8s probe : https://github.com/skydive-project/skydive/blob/master/etc/skydive.yml.default#L129
For example, only enable "cluster", "service", "pod" and "node".

Reply:
Got it, what do you think about:
 - Leave `SKYDIVE_ANALYZER_TOPOLOGY_PROBES` empty by default.
 - Add a template variable for `SKYDIVE_ANALYZER_TOPOLOGY_PROBES` so if like you can add `k8s` at deployment time.
 - And a second template variable for k8s probes. 
   @lebauce do you know the name of the env. var? 
   Should be `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="cluster service pod node"`
   Default also empty.

Reply:
Sounds perfect. Yes, it should be `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES

Reply:
If I tried to start the analyzer with `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES` than I got an `SIGSEGV` error:
```
2019-01-17T14:26:17.666Z	INFO	k8s/probe.go:151 NewProbe	skydive-analyzer-1-r75v5: Probe k8s subprobes [cluster service pod node]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xd19aca]

goroutine 1 [running]:
github.com/skydive-project/skydive/graffiti/graph.(*Indexer).Start(0xc4209349c0)
	/root/go/src/github.com/skydive-project/skydive/graffiti/graph/indexer.go:161 +0x2a
github.com/skydive-project/skydive/topology/probes/k8s.newNamespaceLinker(0xc4206f93e0, 0x21aeb36, 0x3, 0xc420589c80, 0xc, 0xc, 0x432268, 0xc0)
	/root/go/src/github.com/skydive-project/skydive/topology/probes/k8s/namespace.go:60 +0x209
github.com/skydive-project/skydive/topology/probes/k8s.(*Probe).AppendNamespaceLinkers(0xc42088ed00, 0xc420589c80, 0xc, 0xc)
	/root/go/src/github.com/skydive-project/skydive/topology/probes/k8s/probe.go:140 +0x65
github.com/skydive-project/skydive/topology/probes/k8s.NewK8sProbe(0xc4206f93e0, 0xc42088aed0, 0xc4200540b1, 0x3)
	/root/go/src/github.com/skydive-project/skydive/topology/probes/k8s/k8s.go:117 +0xa5d
github.com/skydive-project/skydive/analyzer.NewTopologyProbeBundleFromConfig(0xc4206f93e0, 0xc4201a8a30, 0xc4206f93e0, 0xc420866f60)
	/root/go/src/github.com/skydive-project/skydive/analyzer/probes.go:53 +0x4e6
github.com/skydive-project/skydive/analyzer.NewServerFromConfig(0xc420219fe0, 0x21f31f7, 0x1f)
	/root/go/src/github.com/skydive-project/skydive/analyzer/server.go:282 +0xd5a
github.com/skydive-project/skydive/cmd/analyzer.glob..func1(0x35a0380, 0xc4203b4b10, 0x0, 0x3)
	/root/go/src/github.com/skydive-project/skydive/cmd/analyzer/analyzer.go:49 +0xe8
github.com/skydive-project/skydive/vendor/github.com/spf13/cobra.(*Command).execute(0x35a0380, 0xc4204b6ac0, 0x3, 0x4, 0x35a0380, 0xc4204b6ac0)
	/root/go/src/github.com/skydive-project/skydive/vendor/github.com/spf13/cobra/command.go:603 +0x234
github.com/skydive-project/skydive/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x35a5aa0, 0x0, 0x359ff40, 0xc42062ff58)
	/root/go/src/github.com/skydive-project/skydive/vendor/github.com/spf13/cobra/command.go:689 +0x2d4
github.com/skydive-project/skydive/vendor/github.com/spf13/cobra.(*Command).Execute(0x35a5aa0, 0xc42062ff78, 0xc4200b8058)
	/root/go/src/github.com/skydive-project/skydive/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/root/go/src/github.com/skydive-project/skydive/skydive.go:32 +0x2d
```
I tried with
 - `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="cluster"` 
 - `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="node"`
 - `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="cluster service pod node"`

all got the same error.

Analyzer version:`Skydive Analyzer 0.21.0-f88335bd44dc`

@lebauce do you have some informations about that?

Reply:
Added the changes, wait of feedback from @lebauce. If you leave `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES` empty all works very well.

Reply:
Interesting. This is a probably a bug in the `k8s` probe (cc @hunchback). Can you try with SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="namespace cluster service pod node". If it works, I think we can merge your PR as it

Reply:
With `SKYDIVE_ANALYZER_TOPOLOGY_K8S_PROBES="namespace cluster service pod node"` it works very well.